### ISSUE TITLE
[4.0.0-beta3][ES] Updated version "new in this version" text problem and small typo error. 

### ISSUE THREAD

**Issue by Dragda** [recalbox/recalbox-os#931]
Since installing the 4.0.0 beta 3 version, something weird happens. I have the update option "unstable"  active; and after an update the screen what show me is from the 4.0.0 beta 2 and its fixes.
Today i updated from the buld 79 to the 95 and the error remains the same. If i change it to a "stable" update state, it appears again the message about an update (although it is the last version) and i'm afraid it may reverse my sistem to an old build (Maybe the last stable?).  
This happens in an Raspberry 2B. After a looking in the forum with no result; where i can look the new builds information?. After an update i like to know what is fixed, what not; and what are the news.

In another order, and is an old mistake but overlooked...  In the Spanish language, when you enter in the system configuration menu; at bottom it says "cofiguración de Kodi" when the correct is "configuración de Kodi". One letter is missing on configuration.

**Comment by substring** [comment]
should be fixed in the beta4, we won't make a new release for a typo problem ;)
Anyway, despite the version number is wrong, the features announced are the good ones
We'll make sure beta4 is fine :)

Concerning the translation problem, you can edit https://github.com/recalbox/recalbox-emulationstation/blob/rb-4.0.X/locale/lang/es/LC_MESSAGES/emulationstation2.po#L202 and submit the change in the 4.0 branch. If not, ok i'll do it

**Comment by Dragda** [comment]
Yeah, of course it's a thing to fix in the next beta. No hurries :)
The announced features in the unstable ones are the same of the beta2 to me; but ok, i'll wait to the b4 to see if it's fixed.

I changed the typo error, and added the translation of the new quick menu: "fast shutdown system" ;) 
I looked for more typo errors, but i think that was the only one.

**Comment by substring** [comment]
indeed the changelog is totally messed out for this release lol anyway, the one found on github is the right one

thank you for you contribution concerning translations, that's very appreciated :)
don't forget to refer to this issue when you make your PR for ES, so that we know when to close the issue :)
